docs/topics/logging.txt

```diff
@@ -728,23 +728,57 @@ logging module.
 Django's default logging configuration
 ======================================
 
-By default, Django configures the following logging:
```

Comment:
I would add this content where you currently have "The above behavior can be easily scrutinized:" as it doesn't seem like introductory material.